themes/jb/layouts/partials/head.html

```diff
@@ -5,18 +5,19 @@
   {{ partial "meta.html" . }}
 
   {{ $sass := resources.Get "css/main.sass" }}
+  {{ $video := resources.GetRemote "https://vjs.zencdn.net/7.18.1/video-js.min.css" }}
   <!-- Defining options that production & development should have -->
   {{ $options := (dict "enableSourceMap" true ) }}
   {{ $style := "" }}
 
   {{ if hugo.IsProduction }}
     {{ $options = merge $options (dict "outputStyle" "compressed") }}
     {{ $style = $sass | toCSS $options | minify | fingerprint }}
-
   {{ else }}
     {{ $options = merge $options (dict "outputStyle" "expanded") }}
     {{ $style = $sass | toCSS $options }}
   {{ end }}
   <link rel="stylesheet" href="{{ $style.RelPermalink }}">
+  <link rel="stylesheet" href="{{ $video.RelPermalink }}">
```

Comment:
Adding the `integrity` attribute referenced [here](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/445/files#r990674593).

```suggestion
  <link rel="stylesheet" href="{{ $video.RelPermalink }}" integrity="{{ $video.Data.Integrity }}">
```